src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7704,6 +7702,8 @@ public static class InternalSymbolicLinkLinkCodeMethods
 
         private const uint IO_REPARSE_TAG_MOUNT_POINT = 0xA0000003;
 
+        private const uint IO_REPARSE_TAG_APPEXECLINK = 0x8000001B;
```

Comment:
Can you confirm that it is public and no compliance issue here?

Reply:
It's defined in at least version 10.0.18362.0 of Windows SDK within winnt.h.  It looks like docs.microsoft.com doesn't update frequently enough to have it.